tajo-core/src/main/java/org/apache/tajo/engine/parser/SQLAnalyzer.java

```diff
@@ -491,14 +506,16 @@ public WindowSpec buildWindowSpec(SQLParser.Window_specificationContext ctx) {
     WindowFrameStartBoundType boundType = null;
     if (checkIfExist(context.UNBOUNDED())) {
       boundType = WindowFrameStartBoundType.UNBOUNDED_PRECEDING;
-    } else if (checkIfExist(context.unsigned_value_specification())) {
+    } else if (checkIfExist(context.PRECEDING())) {
       boundType = WindowFrameStartBoundType.PRECEDING;
+    } else if (checkIfExist(context.FOLLOWING())) {
+      boundType = WindowFrameStartBoundType.FOLLOWING;
     } else {
       boundType = WindowFrameStartBoundType.CURRENT_ROW;
```

Comment:
Oh, I wrongly set the default value of start bound.
I think you mean that the default value of start bound should be UNBOUNDED PRECEDING, right?
I'll change the default start boundType to UNBOUNDED PRECEDING.